iceoryx_binding_c/source/c_notification_info.cpp

```diff
@@ -54,6 +60,11 @@ iox_user_trigger_t iox_notification_info_get_user_trigger_origin(iox_notificatio
     return self->getOrigin<UserTrigger>();
 }
 
+iox_client_t iox_notification_info_get_client_origin(iox_notification_info_t const self)
+{
+    return self->getOrigin<UntypedClient>();
+}
+
```

Comment:
Please add tests for these two new functions to `test_notification_info.cpp`.

Reply:
The notification info is tightly coupled to the waitset and therefore it does not make sense to test them separately since one requires a waitset to test it.

I already verified those functions in the waitset and later we may have to clean up those tests a bit.